src/worker-helpers.js

```diff
@@ -45,7 +51,8 @@ export function findESLintDirectory(modulesDir, config, projectPath) {
   let locationType = null
   if (config.useGlobalEslint) {
     locationType = 'global'
-    const prefixPath = config.globalNodePath || getNodePrefixPath()
+    const configGlobal = config.globalNodePath && cleanPath(config.globalNodePath)
```

Comment:
__*Minor Suggestion*__

You could avoid the awkward `&&` here if `cleanPath` was changed to handle a falsey argument by just returning it unchanged or returning `null` or something.  It might not be a bad idea to add such a check to it no matter what, although I'm not sure what it does now with falsey values, maybe it already does.

Reply:
I'm guessing it would have thrown a `TypeError` somewhere in there. If it's falsy I just made it return an empty string. (`null` would have caused errors when used in `path.*` functions.)